background/main.ts

```diff
@@ -869,29 +866,75 @@ export default class Main extends BaseService<never> {
   }
 
   async enrichActivities(addressNetwork: AddressOnNetwork): Promise<void> {
-    const accountsToTrack = await this.chainService.getAccountsToTrack()
```

Comment:
Why did you remove this?

Reply:
I didn't remove it. I just moved it into each individual "enrichITXActivity" and "enrichETXActivity" functions, since it wasn't needed in overall "enrichActivities"